### ISSUE TITLE
[Bug] Server does not support fairseq models

### ISSUE THREAD

**Issue by DenysVuika** [idiap/coqui-ai-TTS#25]
### Describe the bug

It is not possible to run the server.py with the fairseq

### To Reproduce

```sh
docker run --rm -it -p 5002:5002 --platform linux/amd64 --entrypoint /bin/bash ghcr.io/idiap/coqui-tts
python3 TTS/server/server.py --model_name "tts_models/crh/fairseq/vits"
```

### Logs

```shell
File "/root/TTS/server/server.py", line 111, in <module>
    synthesizer = Synthesizer(
  File "/root/TTS/utils/synthesizer.py", line 96, in __init__
    self._load_tts(tts_checkpoint, tts_config_path, use_cuda)
  File "/root/TTS/utils/synthesizer.py", line 186, in _load_tts
    self.tts_config = load_config(tts_config_path)
  File "/root/TTS/config/__init__.py", line 85, in load_config
    ext = os.path.splitext(config_path)[1]
  File "/usr/lib/python3.10/posixpath.py", line 118, in splitext
    p = os.fspath(p)
TypeError: expected str, bytes or os.PathLike object, not NoneType
```

### Environment

```shell
Docker container
```

**Comment by eginhard** [comment]
Yes, the server doesn't support the fairseq models yet (also see https://github.com/coqui-ai/TTS/discussions/3211), also without the Docker: `tts-server --model_name "tts_models/crh/fairseq/vits"`

**Comment by erew123** [comment]
FYI Facebook/Fairseq on Windows stopped on 12.2 of Fariseq back in 2022 and it doesn't work with versions of Python beyond 3.10, so its kind of dead in the water for future updates.

Fairseq version 2, Facebook say they will not be supporting Windows platforms https://github.com/facebookresearch/fairseq2?tab=readme-ov-file#installing-on-windows

I have some kludge versions of fairseq, aka version 12.3 if you will, that work on Windows, and there is someone who has updated Fariseq 12.x beyond that https://github.com/VarunGumma/fairseq, however, I couldn't compile it on Windows currently, due to a bug in Pytorch that will be resolved in later versions of Pytorch (so I read hunting the forums on the issues).

**Comment by eginhard** [comment]
> FYI Facebook/Fairseq on Windows stopped on 12.2 of Fariseq back in 2022 and it doesn't work with versions of Python beyond 3.10, so its kind of dead in the water for future updates.

Coqui doesn't depend on fairseq at all, so this is not an issue. It just loads the trained fairseq models and makes them compatible with Coqui's format.
